app/src/shared/components/Button/newButton.pcss

```diff
@@ -0,0 +1,259 @@
+.root {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding: 0;
+  background-color: transparent;
+  border: 1px solid;
+  border-radius: 4px;
+  font: var(--sans);
+  font-weight: var(--medium);
+  letter-spacing: 0;
+  text-align: center;
+  transition: 200ms ease-in-out;
+  transition-property: border-color, background-color, color, transform;
+  transform: transale3d(0, 0, 0);
```

Comment:
A typo slipped in from my suggestion: `translate3d` → `transale3d`. Also, each cascading `transform: scale(…)` should also have the `translate3d` portion. Otherwise 3d acceleration gets lost in interaction.

Reply:
> Question: is the button 35px in the designs as well? Maybe go with `um` unit?

I checked from the designs and indeed we no more scale button size responsively. I'll fix it to use `um` and it will work as intended.